src/ResourceManager/Resources/Commands.Resources/Models.Authorization/AuthorizationClient.cs

```diff
@@ -183,10 +192,9 @@ public List<PSRoleDefinition> FilterRoleDefinitionsByCustom(string scope, bool s
             result.AddRange(AuthorizationManagementClient.RoleDefinitions.List(
                     scope,
 #if !NETSTANDARD
-                    parameters: new ListDefinitionFilterParameters { AtScopeAndBelow = scopeAndBelow })
-                .RoleDefinitions
+                    scopeAndBelow ? new ODataQuery<RoleDefinitionFilter>(filter => filter.AtScopeAndBelow()) : null)
```

Comment:
when we referenced the new package, we will execute the 'if' condition,  so we changed the code, I have sent you the email for details.